sklearn/neural_network/tests/test_mlp.py

```diff
@@ -662,6 +662,17 @@ def test_warm_start():
             clf.fit(X, y_i)
 
 
+def test_warm_start_full_iteration():
+    X = X_iris
+    y = y_iris
+    max_iter = 3
+    clf = MLPClassifier(hidden_layer_sizes=2, solver='sgd',
+                        warm_start=True, max_iter=max_iter)
+    clf.fit(X, y)
```

Comment:
```suggestion
    clf.fit(X, y)
    assert max_iter == clf.n_iter_
```

Reply:
That is a bit hard to read since it is the same code, do you mean something in the vein of
```
clf.fit(X, y)
assert max_iter == clf.n_iter_
clf.fit(X, y)
assert max_iter == clf.n_iter_
```
That is, testing two times that `max_iter` has been reached?

Reply:
Done that with the last set of commits pushed.